File: backend/app/models/modulo.py
from app.extensions import db

class Modulo(db.Model):
    __tablename__ = 'modulo'

    cod_modulo = db.Column(db.Integer, primary_key=True)
    cod_curso = db.Column(db.Integer, nullable=False)
    titulo_modulo = db.Column(db.String(256), nullable=False)
    descripcion_modulo = db.Column(db.Text)

File: frontend/src/types/modulo.ts
export interface Modulo {
  cod_curso: number;
  cod_modulo: number;
  titulo_modulo: string;
  descripcion_modulo: string;
}
export interface CreateModuloData {
  cod_curso: number;
  titulo_modulo: string;
  descripcion_modulo?: string;
}
